### DISCUSSION TITLE
"Task lists! Give feedback" notification

### DISCUSSION THREAD

**Discussion by CreepyGnome** [community/community#5296] [Projects and Issues] [answered]
Few things, this is vague as they are not new we have been using checkboxes in our PR templates for a while now, and this thing just pops up on the PR without any context or more details as to what you expect and clicking on it just dumps you in the root of the issues-feedback 

![image](https://user-images.githubusercontent.com/935891/130867253-42fb140e-b3c3-4080-bd6a-a89274d14e38.png)

So I guess my feedback is this prompt is confusing as hell. It is "yelling" at me "Task lists!" without any real context about WHY it is, and then asking me to give feedback with a link that takes me to a list of all issues feedback, which is more confusing.

**Comment by fivekWBassMachine** [comment]
Came here the same way. The lists are great but everything else than new...

**Comment by TicClick** [comment]
seeing you here I would say that the feedback request has worked. jest aside, it could request your opinion on what should be improved for the lists as we know them.

**Reply by Tarcontar** [reply]
Templates would be nice for recurring tasklists in PRs!

**Reply by Kixunil** [reply]
This seems to be already available - I saw exactly this when contributing to lightningnetwork/lnd

**Reply by CreepyGnome** [reply]
I am sure the feedback request did not work as intended as I am giving them feedback about how they are requesting feedback primarily.  Yeah so if they asked a question and then feedback that would be great, not just yell at us with "Task lists!" as if it's exciting and new, which is not exciting and not new.  I doubt anyone at GitHub will even read let alone respond to this "feedback".

Yes, templates are not new so they are nice and they have had task lists (aka just checking boxes) available for a while.  Not sure why they are asking for feedback without context on an old feature.

**Comment by Kixunil** [comment]
Agree. Let me take this opportunity to express what I would like to see: a way to reference that a PR checks a box in the tasklist without closing the whole issue and perhaps close the issue once all are checked. I imagine something like `#123/0` to refer to topmost item in the list, `#123/1` to refer to the next one etc. Or perhaps something like `#123/"write tests"`, `#123/"add tests to CI"` matching the prefix of the task.

**Reply by CreepyGnome** [reply]
That would probably be best as its own feature request/feedback instead of an answer also not sure why "feedback" is really just built on their discussions having an "answer" to feedback seems stupid as its feedback to the company about something... anyways rant over... ;) 

to get best visibility would make your answer here its own dedicated feedback/feature request.

**Reply by Kixunil** [reply]
Yeah, didn't notice there are other issues for it so it seems there isn't a dedicated issue for all task-related things.

**Comment by shayfrendt** [comment, marked as answer]
👋🏻  Ohai! Thanks for starting this discussion @CreepyGnome. 🙇🏻 

Looks like it's gotten a bunch of upvotes, so it seems others have the same feels. 🕺🏻 

You're totally right about the feedback call to action being vague: tasks lists in markdown have existed [since 2014](https://github.blog/2014-04-28-task-lists-in-all-markdown-documents/). ✅ 

So why would we be asking about them now? 

The missing context there is that earlier this year we [released](https://github.blog/changelog/2021-06-23-whats-new-with-github-issues/) some improvements to task list functionality in issues to make it easier to break related work down. And so that's what led to the (excited, as you've noted) ask for feedback. 🙂 

Thanks to everyone that did share their thoughts, we've got some great feedback on how to improve the feature even more. 🚀 

We've removed this call to action as it's run its course! 

And next time, we'll try to provide more context on what's changed in order to frame the request for feedback. 👍🏻

**Reply by CreepyGnome** [reply]
I would say do better than "try" next time, the feedback button without the context of what you want feedback on and why, like specifics, and also just dropping you in your github.com/feedback is worthless period.  Maybe it should take you to a specific discussion page that is about the feedback you are requesting that explains its.  

Better yet stop using discussions for feedback in its current form, make a feedback specific feature so that you can have a single GitHub created "post" where instead of "Answers" it getts "Feedback" and feedback responses cannot be marked "answered" but they can each be upvoted.  So you are close with discussions so making a Feedback specific feature based on it would not be to difficult probably.
